### PR TITLE
fix: translation bug

### PR DESCRIPTION
**Pull request by Harshwardhan18**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -74,6 +74,8 @@ export class AppComponent implements OnInit, OnDestroy {
           //   title: "Website Language",
           //   msg: `Are you sure you want to change the language to ${this.language.name}?`,
           // };
+        } else {
+          this.router.navigate([], { relativeTo: this.route });
         }
```
Comment (Manzurkds) [inline src/app/app.component.ts:79]: @Harshwardhan18 Why was this added?

**Review by Manzurkds — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (Harshwardhan18) [reply src/app/app.component.ts:79]: @Manzurkds it removes the query params
